docs/CI-CD/dockerByPR.yml

```diff
@@ -0,0 +1,98 @@
+name: DockerCI-PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+    paths-ignore:
+      - 'docs/**'
+      - '.github/**'
+      - '**.md'
+      - 'saveenv.sh'
+
+jobs:
+
+  docker-build-push:
+
+    runs-on: ubuntu-latest
+    env:
+      #  DOCKER_REPO  - the docker repository name in the form: organization/repository
+      DOCKER_REPO: 'retaildevcrew/helium-csharp'
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Login to registry
```

Comment:
I would add a step that checks for the secrets and fails with a friendly and informative error message

Reply:
We should probably go ahead and do the build step and just display a message that we can't push because the secrets aren't set. If the build passes, then the action returns green with a message that the image wasn't pushed because the secrets are missing